region/maridia/inner-green/West Sand Hall.json

```diff
@@ -623,6 +666,79 @@
       ],
       "note": "It is possible to cross this segment with nothing. Cancel spin before hitting the sand, then get a good jump off the sand in multiple places."
     },
+    {
+      "link": [2, 5],
+      "name": "West Sand Hall Tricky Speedy Jump (Right to Left)",
+      "notable": true,
+      "entranceCondition": {
+        "comeInRunning": {
+          "minTiles": 10.4375,
+          "speedBooster": true
+        }
+      },
+      "requires": [
+        "canSuitlessMaridia",
+        "canPlayInSand",
+        "canTrickyDashJump"
+      ],
+      "note": [
+        "Run into the room using between 11 and 12 tiles of runway in the other room (extra run speed between $2.D and $3.1).",
+        "Jump somewhat soon after entering the room, and land on the pillar just past the last Evir.",
+        "To get a controlled landing, it can help to turn around while still in spin, then break spin immediately after."
```

Comment:
Turning around early and drifting to the platform while slowing down is funny.  Then break spin to stop instantly.